modules/azure/storage_account_private/main.tf

```diff
@@ -43,8 +43,14 @@ resource "azurerm_private_endpoint" "private_endpoint" {
     subresource_names              = ["blob"]
   }
 
-  # Should be deployed by Azure policy
-  lifecycle {
```

Comment:
Does this not conflict with existing deployments where a policy creates the DNS zone?

Reply:
Not 100% sure to be honest, I think it might work out because they changed the provider to allow configuring DNS zones using the block. We could create a major version to make sure.

Reply:
I would not create a major version, because it is a mono repo. I would instead test it out, to make sure this is both supported. And/or if there is no solution to support both, check what the best practice is. Unfortunately lifecycle items cannot depend on variables/conditions (yet).

Reply:
There is only one use of the module as far as I can see, and the project is discontinued. Seems like overkill.

Reply:
Could be, but it doesn't change the fact that there are other similar implementations in modules that eventually would require setting this by policy (ignore changes) OR implementing it in the module. I will approve it for now, but keep this in mind :)